proto/greptime/v1/row.proto

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package greptime.v1;
+
+option java_package = "io.greptime.v1";
+option java_outer_classname = "RowData";
+option go_package = "github.com/GreptimeTeam/greptime-proto/go/greptime/v1";
+
+import "greptime/v1/common.proto";
+
+message Rows {
+  repeated ColumnSchema schema = 1;
+  repeated Row rows = 2;
+}
+
+message ColumnSchema {
+  string column_name = 1;
+  ColumnDataType datatype = 2;
+  SemanticType semantic_type = 3;
+}
+
+message Row {
+  repeated Field fields = 1;
```

Comment:
Could we choose another name? A field is used to descript a column in a schema.

Candidates:
## Value
```
message Row {
  repeated Value values = 1;
}
message Value {
  oneof value_data {}
}
```

## Datum
```
message Row {
  repeated Datum datums = 1;
}

message Datum {
  oneof value {}
}
```